packages/ol/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@react-email/ol",
+  "version": "0.0.1",
+  "description": "An ordered list component to help build emails",
+  "sideEffects": false,
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --dts --external react",
+    "dev": "tsup src/index.ts --format esm,cjs --dts --external react --watch",
+    "lint": "TIMING=1 eslint src/**/*.ts* --fix",
+    "clean": "rm -rf dist",
+    "test": "jest",
+    "test:watch": "jest --watch"
```

Can you add the format scripts here as well?

```suggestion
    "test:watch": "jest --watch",
    "format:check": "prettier --check \"**/*.{ts,tsx,md}\"",
    "format": "prettier --write \"**/*.{ts,tsx,md}\""
```

---

Done!